tests/WebAppCallsMicrosoftGraph/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using Microsoft.Graph;
+
+namespace WebAppCallsMicrosoftGraph.Pages
+{
+    public class IndexModel : PageModel
+    {
+        private readonly ILogger<IndexModel> _logger;
+        private readonly GraphServiceClient _graphServiceClient;
+
+        public IndexModel(ILogger<IndexModel> logger, GraphServiceClient graphServiceClient)
```

Comment:
`IGraphServiceClient`?
```suggestion
        public IndexModel(ILogger<IndexModel> logger, IGraphServiceClient graphServiceClient)
```

Reply:
Same thing. @darrelmiller ?

Reply:
Yup.